src/featureFlag.ts

```diff
@@ -0,0 +1,62 @@
+import * as ld from 'launchdarkly-node-client-sdk';
+import { getCurrentBranch } from './gitUtils';
+
+let launchdarklyClient: LaunchdarklyClient;
+
+class LaunchdarklyClient {
```

Comment:
I didn't import the AFE feature flag package because it requires adding a custom field in root package.json that links to the project in LD, which means one product is mapped to one LD project. As a result, we need to create a FF with the same name across multiple LD projects to make it work across different products.

This package is only using one [FF](https://app.launchdarkly.com/atlassian-frontend/production/features/peer-dependency-enforcement_operational/targeting) to control the check.